sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -372,6 +372,8 @@ def parse_connection_str(conn_str, credential, keyword_args):
             }
         except KeyError:
             credential = conn_settings.get("sharedaccesssignature")
+            # if "sharedaccesssignature" in conn_settings:
+            #     credential = AzureSasCredential(conn_settings['sharedaccesssignature'])
```

Comment:
Do we need this wrapper?

Reply:
I'm somewhat undecided....
I've left it as a TODO for me to revisit before GA.